### PR TITLE
Create snipe-it proxy conf

### PR DESCRIPTION
**Pull request by junkman690**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

**Comment by junkman690** [comment]
This should also close issue #262

File: snipe-it.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
+
+        include /config/nginx/proxy.conf;
+        include /config/nginx/resolver.conf;
+        set $upstream_app snipe-it;
+        set $upstream_port 443;
+        set $upstream_proto https;
```
Comment (Roxedus) [inline snipe-it.subdomain.conf.sample:37]: Is there any reason it talks to the container over https?

Comment (junkman690) [reply snipe-it.subdomain.conf.sample:37]: @Roxedus I honestly can't recall why I set it to https. http should also work fine.

**Review by nemchik — CHANGES_REQUESTED** [review]
The proxy templates have been updated with new descriptive comments and auth lines. Please update your PR accordingly.

Ref:
https://github.com/linuxserver/reverse-proxy-confs/blob/master/_template.subdomain.conf.sample
https://github.com/linuxserver/reverse-proxy-confs/blob/master/_template.subfolder.conf.sample

**Pull request closed without merging**